common/templates/context_funcs.go

```diff
@@ -1335,3 +1336,122 @@ func (c *Context) tmplEditNickname(Nickname string) (string, error) {
 
 	return "", nil
 }
+
+func (c *Context) tmplSort(slice []interface{}, sortargs ...interface{}) (interface{}, error) {
+	if c.IncreaseCheckCallCounterPremium("sortfuncs", 1, 3) {
+		return "", ErrTooManyCalls
+	}
+
+	var dict SDict
+	var err error
+	switch len(sortargs) {
+	case 0:
+		input := make(map[string]interface{}, 3)
+		input["reverse"] = false
+		input["subslices"] = false
+		input["emptyslices"] = false
+
+		dict, err = StringKeyDictionary(input)
+		if err != nil {
+			return "", err
+		}
+	case 1:
+		dict, err = StringKeyDictionary(sortargs[0])
+		if err != nil {
+			return "", err
+		}
+	default:
+		dict, err = StringKeyDictionary(sortargs...)
+		if err != nil {
+			return "", err
+		}
+	}
+
+	var numberSlice, stringSlice, timeSlice, csliceSlice, mapSlice, defaultSlice, outputSlice []interface{}
+
+	for _, v := range slice {
+		switch t := v.(type) {
+		case int, int64, float64:
```

Comment:
Lets not mix float64 with int64 as float64 only has integer level precision up to 2^53